src/rpcblockchain.cpp

```diff
@@ -190,10 +190,10 @@ Value getdifficulty(const Array& params, bool fHelp)
             "getdifficulty\n"
             "Returns the difficulty as a multiple of the minimum difficulty.");
 
-    //Object obj;
-    //obj.push_back(Pair("proof-of-work",        GetDifficulty()));
-    //obj.push_back(Pair("proof-of-stake",       GetDifficulty(GetLastBlockIndex(pindexBest, true))));
-    return GetDifficulty(GetLastBlockIndex(pindexBest, true));
+    Object obj;
+    obj.push_back(Pair("proof-of-work",        GetDifficulty()));
```

Comment:
Whats with the strange spacing?

Reply:
I don't know, it's like this all over the place so I followed the "theme"

Reply:
Think you can fix it for just the lines that you touched and put out another PR? First step in fixing the trend.

Reply:
Did the three files, see a6947b6